mvlearn/__init__.py

```diff
@@ -4,3 +4,6 @@
 import mvlearn.cotraining
 import mvlearn.datasets
 import mvlearn.utils
+import mvlearn.jive
```

Comment:
It appears here and other files that the newline at the end of the file is being removed. This will result in a style check fail as far as I know. Also, in master the version is 0.0.2

Reply:
So should I take out this line?

Reply:
You want a newline at the end of every file (I believe) else style check will fail.